Automattic/wp-calypso
# Copy site: Fix copy site when using various payment methods

Fixes https://github.com/Automattic/dotcom-forge/issues/1958#issue-1621351950

## Proposed Changes

When a user tries to copy a site using payment methods other than credits, the copy site doesn't complete successfully.
This pr fix that.

## Testing Instructions



### Test 1

1.  Make sure you have a test user account in place.
2. Ensure that you have followed the instructions in the fieldguide (search for Testing the Store. I've not pasted the URL, as the script that handles the URLs is not working for the fieldguide for me)
3. Make sure that you visit the secret store and find the credentials with the following name: PayPal Checkout
4. Copy a site.
5. Use PayPal as a payment method, and checkout.
6. Ensure that the copy site process completes successfully.

### Test 2

1.  Make sure you have a test user account in place.
2. Ensure that you have disabled the cookie you set up in the previous test
4. Copy a site using credits.
6. Ensure that the copy site process completes successfully.

## Pre-merge Checklist
